packages/kernel-6.12/config-bottlerocket

```diff
@@ -48,6 +48,35 @@ CONFIG_FB=y
 CONFIG_FB_EFI=y
 CONFIG_FRAMEBUFFER_CONSOLE_DEFERRED_TAKEOVER=y
 
+# Do not allow writes to a block device if a filesystem is mounted on it.
+# CONFIG_BLK_DEV_WRITE_MOUNTED is not set
+
+# Bottlerocket does not hibernate
+# CONFIG_HIBERNATION is not set
+
+# Do not permit processes to push characters into their controlling tty device
+# CONFIG_LEGACY_TIOCSTI is not set
```

Comment:
Does `apiclient exec` still work with this change?

Reply:
Yes, it does.